scripts/coordinator/src/modules/core/index.ts

```diff
@@ -26,6 +26,11 @@ export class CoreModule implements ManagerModule {
     return this._config;
   }
 
+  private _lastRun: number = 0;
+  get lastRun(): number {
```

Comment:
We would extract this method to ManagerModule since it appears in two different instances of it (CoreModule, PumpModule)